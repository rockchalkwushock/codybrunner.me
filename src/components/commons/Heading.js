import styled from 'styled-components'

const Heading = styled.h1`
  font-size: ${({ size }) => size};
  margin: unset;
  padding-bottom: 1rem;
  text-align: center;
`

Heading.defaultProps = {
  size: '1rem'
}

export default Heading
