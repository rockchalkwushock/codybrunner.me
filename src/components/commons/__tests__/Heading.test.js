import React from 'react'
import { shallow } from 'enzyme'

import Heading from '../Heading'

describe('Component: <Heading />', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<Heading />)
  })

  test('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
