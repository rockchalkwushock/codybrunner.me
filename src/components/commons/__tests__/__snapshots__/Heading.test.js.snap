// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Heading /> should render correctly 1`] = `
<h1
  className=""
/>
`;
